### ISSUE TITLE
iPad: Can't open posts in the Reader

### ISSUE THREAD

**Issue by kean** [wordpress-mobile/WordPress-iOS#23652]
The user reports says that since the last (TestFllight) update (with a sidebar) that can't open any posts in the Reader. Once they tap on a post title, the app becomes unresponsive.

App version: 25.4
Device model: iPad Air (4th gen)
OS version: 17.6.1
Type of site (Simple, Atomic, Jetpack, or self-hosted): all

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by kean** [comment]
I can't reproduce it.

**Comment by dcalhoun** [comment]
Noting that I experienced sporadic, momentary unresponsiveness with the sidebar navigation on an iPhone SE (p1728411749700589/1728406178.542969-slack-C07NPKBC88L).  I have not identified steps to reproduce.

**Comment by Siphompeks** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by kean** [comment]
I checked the logs from one of the instances and nothing suspicious there except for maybe this:

```
2024-10-24 05:46:25:767 deletePostsWithNoTopic, deleting topicless post: <ReaderPost: 0x303a2c500> (entity: ReaderPost; id: 0x9000f667de0c46eb <x-coredata://AD87435E-B29D-43DE-A395-3B90E0307985/ReaderPost/p39>; data: <fault>)
2024-10-24 05:46:25:772 Deleting topic: (null)
```

**Comment by kean** [comment]
**RCA:** there is an issue with a deprecated `pushFullscreenViewController` code in 25.4 when "Reduce Motion" is enabled.
**Workaround:** disable "Reduce Motion" in "Accessibility" / "Motion" in the Apple Settings app.

**Comment by kean** [comment]
Fixed in 24.5.2 https://github.com/wordpress-mobile/WordPress-iOS/pull/23731 and fixed in `trunk` before by removing the "reduce motion" code. It now follows the standard system behavior.
